### ISSUE TITLE
Accessibility/GitHub (Combined): Evaluate Ingestion of GitHub A11y Issue Data Into BigQuery

### ISSUE THREAD

**Issue by pjhill** [department-of-veterans-affairs/va.gov-team#32002]
# Context

We already have data coming into BQ from GH issues [here](https://github.com/department-of-veterans-affairs/testing-tools-team-dashboard-data).

## User Story

_We currently have GitHub A11y issue data being ingested into BigQuery and much of what we need exists in the data we have. This data should be evaluated to ensure that we have what we need to be able to track our data points properly, and that the ingestion of this data is being triggered regularly enough to provide us the data we are looking for. Given that we want to evaluate GitHub data from more than one context, we need to ensure that GitHub data ingestion includes everything we need for other GitHub related visualizations. The GitHub related metrics we'll need to track are:_

### A11y
* Current running total of a11y issues filed against the product, charted by date.
* Breakdown of current running total (most recent date) into what kinds of a11y violations they are.
* Breakdown of current running total (most recent date) into launch-blocking vs. non-launch-blocking
### General GH
* How many defects are associated with a product?
* How many code additions and deletions are associated with a product over time?
* What is the rate of change a product experiences over time?
* How many pull requests associated with a product have been merged over a given time? How many pull requests associated with a product are open over time?

## Tasks
- [ ] Evaluate current GitHub a11y issue data ingestion mechanism to see if changes need to be made to account for metrics we cannot currently track.
- [ ] Data should be ingested for all applications in one data set, so a Platform-level overview can be given. There should be a team and application associated with each row, so the charts can be properly filtered down to a single application/team view in Domo.
- [ ] Adjust BigQuery Schemas and/or create new BigQuery tables as needed to meet the needs of our new metrics.
- [ ] Evaluate our current trigger that updates this data. If deemed more appropriate, it can be triggered on publishing of a new release or a cron can be set up to ingest this data daily.
- [ ] Create Domo data connections to BQ that pull these data sets to Domo to be ready for cards.
- [ ] Start filling in DOMO cards in the Product Health outline dashboard [here](https://va-gov.domo.com/page/1950369682).

## Acceptance Criteria

- [ ] _Ingestion mechanism provides all the data we need to produce our desired GitHub A11y Issue metrics in usable formats._
- [ ] _Data is set up to automatically ingest at an acceptable interval._
- [ ] _Domo has data connections for all the Contract test data that can be used by our cards._

**Comment by noahgelman** [comment]
I looked over the github data being extracted in BigQuery and looking at the needs in this ticket and I feel that the data currently provided should have everything we need except for possibly comments. Comment data could include author, created date, value, etc. Adding/deleting a comment does update the `updated_at` value however. Current data are things like ticket number, labels, created/updated times, creator, assignees, etc.

Example json data from github in BigQuery: https://api.github.com/repos/department-of-veterans-affairs/va.gov-team/issues/32578

Currently working on a cards in domo to display number of open issues over time and closed tickets over time

**Comment by pjhill** [comment]
Discussion re: what comment data might contribute to --
* Average number of comments
* Average time between comments (implications as to activity / persistence of involvement)

**Comment by pjhill** [comment]
No changes to ingestion were required.
